src/sentry_string.c

```diff
@@ -119,19 +127,39 @@ sentry__string_from_wstr_n(const wchar_t *s, size_t s_len)
 wchar_t *
 sentry__string_to_wstr(const char *s)
 {
-    int len = MultiByteToWideChar(CP_UTF8, 0, s, -1, NULL, 0);
+    const int len = MultiByteToWideChar(CP_UTF8, 0, s, -1, NULL, 0);
+    if (len <= 0) {
+        return NULL;
+    }
     wchar_t *rv = sentry_malloc(sizeof(wchar_t) * (size_t)len);
-    if (rv) {
-        MultiByteToWideChar(CP_UTF8, 0, s, -1, rv, len);
+    if (!rv) {
+        return NULL;
     }
+    MultiByteToWideChar(CP_UTF8, 0, s, -1, rv, len);
     return rv;
 }
+
+wchar_t *
+sentry__string_clone_wstr(const wchar_t *str)
+{
+    if (!str) {
+        return NULL;
+    }
+    size_t str_len = wcslen(str);
```

Comment:
(nit) some more const
```suggestion
    const size_t str_len = wcslen(str);
```